src/fn/check-git.js

```diff
@@ -0,0 +1,35 @@
+const git = require('../lib/git-exec');
+const { info, warn, error } = require('../lib/log');
+const userPrompt = require('../lib/user-prompt');
+const environment = require('../lib/environment');
+
+module.exports = {
+  requiresInstance: false,
+  execute: async () => {
+    if (!environment.isProduction() && environment.instanceUrl) {
```

Comment:
So if no `instanceUrl` is set we'll run the check? I'm wondering if we should skip it in this case, eg:

```suggestion
    if (!environment.isProduction() || !environment.instanceUrl) {
```

Can you think of an example of when we would want to check git but no instanceUrl is given?

Reply:
Maybe is unlikely but if the user call explicitly the actions to execute, eg.:

    $ mmconf check-git compress-pngs fetch-csvs-from-google-drive ...

And `check-git` is in that list explicity but no URL was provided (there are tasks that don't need to connect with the instance), that would be the case.

Reply:
Ok. We can always change it later if it's too annoying, but yes if they explicitly request it it should run!